src/webots/gui/WbUpdatedDialog.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright 1996-2021 Cyberbotics Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <QtCore/QVariant>
+#include <QtWidgets/QApplication>
+#include <QtWidgets/QGroupBox>
+#include <QtWidgets/QLabel>
+#include <QtWidgets/QPushButton>
+
+#include "WbUpdatedDialog.hpp"
+
+WbUpdatedDialog::WbUpdatedDialog(QWidget *parent) : QDialog(parent) {
+  setWindowTitle(tr("Welcome to Webots R2022a"));
+
+  QPixmap webotsLogo("images:webots.png");
+  QLabel *image = new QLabel(this);
+  image->setGeometry(QRect(16, 15, 50, 50));
+  image->setPixmap(webotsLogo);
+  image->setScaledContents(true);
+
+  QLabel *labelThanks = new QLabel(this);
+  labelThanks->setGeometry(QRect(75, 30, 330, 30));
+  labelThanks->setText(tr("<b>Thank you for using Webots R2022a.</b>"));
+  labelThanks->setStyleSheet("font-size: 15px;");
+  labelThanks->setAlignment(Qt::AlignLeading | Qt::AlignLeft | Qt::AlignTop);
+  labelThanks->setWordWrap(true);
```

Comment:
This is not English, it should be either thanksLabel or simply label:
```suggestion
  QLabel *label = new QLabel(this);
  label->setGeometry(QRect(75, 30, 330, 30));
  label->setText(tr("<b>Thank you for using Webots R2022a.</b>"));
  label->setStyleSheet("font-size: 15px;");
  label->setAlignment(Qt::AlignLeading | Qt::AlignLeft | Qt::AlignTop);
  label->setWordWrap(true);
```